src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -95,6 +99,11 @@ private void ConsoleWriter(string evt)
             }
         }
 
+        private void LogMetricsPublishEvent(LogLevel level, string message)
```

Comment:
This function doesn't appear to be used? This line and the unused commented out line above?